### PR TITLE
This fixes all tests for slice datatype.

### PR DESCRIPTION
**Pull request by student-06**
#36

### CODE REVIEW

File: python/common/org/python/types/Slice.java [hunk 1]
```diff
@@ -240,4 +240,85 @@ public ValidatedValue validateValueTypes() {
...
+    }
+
+    @org.python.Method(
+            __doc__ = "",
+            args = {"other"}
+    )
+    public org.python.Object __imul__(org.python.Object other) {
+        if (other instanceof org.python.types.ByteArray || other instanceof org.python.types.Bytes ||
+                other instanceof org.python.types.List || other instanceof org.python.types.Str ||
+                other instanceof org.python.types.Tuple) {
```
Comment (eliasdorneles) [inline python/common/org/python/types/Slice.java]: Could you refactor this check into a static method `isSequence`, and then reuse the it in `__mul__` ?

**Comment by student-06** [comment]
Hi @eliasdorneles.  I implemented this ```public static boolean isSequence(org.python.Object other)``` in Object class. Really liked your suggestion about definition of sequence type. 
Thank you.

File: python/common/org/python/types/Slice.java [hunk 2]
```diff
@@ -240,4 +240,86 @@ public ValidatedValue validateValueTypes() {
...
+            __doc__ = "",
+            args = {"index"}
+    )
+    public void __delitem__(org.python.Object index) {
+        throw new org.python.exceptions.TypeError("'slice' object does not support item deletion");
+    }
+
+    // @org.python.Method(
+    //     __doc__ = "Return true if "
+    // )
```
Comment (eliasdorneles) [inline python/common/org/python/types/Slice.java]: There are some leftover commented code here, can you please remove? Thanks!

**Comment by eliasdorneles** [comment]
Thanks @student-06 ! :sparkles:

**Pull request merged**